py_ocpi/tokens/v_2_2_1/schemas.py

```diff
@@ -42,6 +42,23 @@ class Token(BaseModel):
     last_updated: DateTime
 
 
+class TokenUpdate(BaseModel):
+    country_code: Optional[CiString(2)]
+    party_id: Optional[CiString(3)]
+    uid: Optional[CiString(36)]
+    type: Optional[TokenType]
+    contract_id: Optional[CiString(36)]
+    visual_number: Optional[String(64)]
+    issuer: Optional[String(64)]
+    group_id: Optional[CiString(36)]
+    valid: Optional[bool]
+    whitelist: Optional[WhitelistType]
+    language: Optional[String(2)]
+    default_profile_type: Optional[ProfileType]
+    energy_contract: Optional[EnergyContract]
+    last_updated: Optional[DateTime]
```

Comment:
I think that last_updated field shouldn't be optional because it's required, according to docs: **_Any request to the PATCH method SHALL contain the last_updated field._**